### PR TITLE
Add pgxn pipeline into develop branch to be able to trigger from ui

### LINKED ISSUE #702: Add pgxn pipeline into develop (i.e. default) branch to be able to trigger it from ui
As defined below, pipeline to be executed manually from ui should be defined in master branch. 
Add update_pgxn_version pipeline into develop branch to be able to run this pipeline from pgxn-citus branch
https://github.community/t/workflow-dispatch-event-not-working/128856/2

### PR DESCRIPTION
**Pull request by gurkanindibay**
Blocked on: https://github.com/citusdata/tools/pull/155

### CODE REVIEW

File: .github/workflows/update-pgxn-version.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    name: Update pgxn configuration
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone Tools branch
+        run: git clone --branch issue_154 https://github.com/citusdata/tools.git
```
Comment (hanefi) [inline .github/workflows/update-pgxn-version.yml]: Needs to be updated after https://github.com/citusdata/tools/pull/155 gets merged.

**Comment by hanefi** [comment]
@gurkanindibay You need to rebase this branch, and update git branch names in the `.yml` file.

**Comment by hanefi** [comment]
This one still needs a rebase

File: .github/workflows/update-pgxn-version.yml (hunk 1 above)
Comment (gurkanindibay) [reply .github/workflows/update-pgxn-version.yml]: Fixed

File: .github/workflows/update_package_properties.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ jobs:
 
       # Runs a single command using the runners shell
       - name: Clone Tools branch
-        run: git clone --branch tools-transition https://github.com/citusdata/tools.git
+        run: git clone --branch v0.8.1 https://github.com/citusdata/tools.git
```
Comment (hanefi) [inline .github/workflows/update_package_properties.yml]:
```suggestion
        run: git clone --depth 1 --branch v0.8.1 https://github.com/citusdata/tools.git
```

File: .github/workflows/update-pgxn-version.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+env:
+  PROJECT_VERSION: ${{ github.event.inputs.project_version }}
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      project_version:
+        description: "Version number to be released e.g 10.0.3"
```
Comment (hanefi) [inline .github/workflows/update-pgxn-version.yml:11]:
```suggestion
        description: "Version number to be released e.g 10.0.3"
        required: true
```

**Review by hanefi — CHANGES_REQUESTED** [review]
I do not like the contents of the file `base-ci.yml`.

Can you rename that file and update all comments in it? I do not really understand its purpose right now, without reading through all the content.

**Comment by gurkanindibay** [comment]
> @gurkanindibay You need to rebase this branch, and update git branch names in the `.yml` file.

Fixed

**Comment by gurkanindibay** [comment]
> I do not like the contents of the file `base-ci.yml`.
> 
> Can you rename that file and update all comments in it? I do not really understand its purpose right now, without reading through all the content.

Removed base-ci since build_package.yml in each branch is enough for package build (all-citus, all-enterprise)

File: .github/workflows/update-pgxn-version.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        required: true
+
+jobs:
+  update_pgxn:
+    name: Update pgxn configuration
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (hanefi) [inline .github/workflows/update-pgxn-version.yml:19]: This one could use a name field.

**Review by hanefi — APPROVED** [review]
One minor comment that can improve GitHub actions UI.

**Pull request merged**